templates/registration/login.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form method="post" action="{% url 'login' %}">
```

Comment:
Verify that the URL name 'login' matches the URL pattern defined in your project's `urls.py`. Ensure consistency with the naming conventions used throughout the project.